app/javascript/app/components/nav/nav-component.jsx

```diff
@@ -2,24 +2,50 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import Link, { NavLink } from 'redux-first-router-link';
 import cx from 'classnames';
-import { Icon } from 'cw-components';
+import environmentalAffairsLogo from 'assets/environmental-affairs-logo.png';
+import SAflag from 'assets/south-africa-flag.png';
 
-import cwLogo from 'assets/cw-logo.svg';
 import styles from './nav-styles.scss';
 
 class Nav extends PureComponent {
   render() {
     const { routes, className } = this.props;
     return (
-      <nav className={cx(styles.navbar, className)}>
-        <Link className={styles.link} to="/">
-          <Icon theme={{ icon: styles.logo }} icon={cwLogo} />
-        </Link>
-        {routes.map(route => (
-          <NavLink className={styles.link} key={route.label} to={route.link} activeClassName={styles.active}>
-            {route.label}
-          </NavLink>
-        ))}
+      <nav className={cx(styles.header, className)}>
+        <div className={styles.navbarContainer}>
+          <div className={styles.navElement}>
+            <Link className={styles.link} to="/">
```

Comment:
I've checked with the designer and this link is not going to the homepage, instead it should go to: https://www.environment.gov.za/

Reply:
👍